Server/controllers/articleController.js

```diff
@@ -37,27 +36,28 @@ const artController = {
     });
   },
   async getSpecific(req, res) {
-    const { email } = req.user; // email of authenticated user
+    // email of authenticated user
     const { id } = req.params;
-  
-    const comment = db.comments;
-
-    const findArticle = await artService.getSpecific(id, email, comment);
+    const artid = parseInt(id, 10);
+    const comment = db.comments.filter((com) => com.artID === artid);
+    const noComment = [];
+    const commToSend = comment.length > 0 ? comment : noComment;
+    const findArticle = await artService.getSpecific(id, commToSend);
     if (!findArticle) return res.status(404).send({ status: 'error', error: 'Article not found, please try another' });
 
     return res.status(200).send({
-      status: 'success',
+      status: 200,
       data: findArticle,
```

Comment:
This should have `message` as a key as well

Reply:
fixed alredy